app/views/shared/_header.html.erb

```diff
@@ -25,6 +25,7 @@
 				<li><%= link_to('Site Settings', '/settings') %></li>
 			<% end %>
 			<li><%= link_to('Favorite Stops', favorites_path) %></li>
+			<li><%= link_to('History', page_path(:page_name => 'history')) %></li>
```

Comment:
The word "History" doesn't convey that this is a user's past issue. What do you think about saying "Reported Issues"?

Reply:
![screen shot 2018-06-19 at 6 22 24 pm](https://user-images.githubusercontent.com/33401891/41629366-be3b1efa-73ee-11e8-8945-18482f393a3b.png)
much better.

Reply:
![screen shot 2018-06-19 at 6 22 39 pm](https://user-images.githubusercontent.com/33401891/41629434-13d58ca6-73ef-11e8-9ac3-d87bbdfe31ab.png)